### PR TITLE
clojure: Recognize ClojureDart files

### PR DESCRIPTION
**Pull request by zachcp**
Added support for ClojureDart (cljd) files in the Clojure Extension

Release Notes:

- N/A

### CODE REVIEW

**Comment by zachcp** [comment]
@cla-bot check

File: extensions/clojure/extension.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id = "clojure"
 name = "Clojure"
 description = "Clojure support."
-version = "0.0.2"
+version = "0.0.3"
```
[the remark this answers is not in the document]
Comment (zachcp) [reply extensions/clojure/extension.toml:4]: ahh. yes

[the remark this answers is not in the document]
Comment (zachcp) [reply extensions/clojure/extension.toml]: Reverted.

**Review by maxdeviant — APPROVED** [review]
Thank you!

**Pull request merged**